### ISSUE TITLE
Bug: Assignment instead of comparison in If

### ISSUE THREAD

**Issue by zeroby0** [gavinlyonsrepo/TM1638plus#22]
Assignment instead of comparison in `else if`.

https://github.com/gavinlyonsrepo/TM1638plus/blob/a102db1dde3cdff316a65b01b40c4d9ad86142a4/src/TM1638plus_Model2.cpp#L89-L95

Should be:

```c++
if (TextAlignment == TMAlignTextLeft) 
{ 
     strcat(TextDisplay, TextLeft);  // %-4X 
} else if ( TextAlignment == TMAlignTextRight) 
{ 
     strcat(TextDisplay, TextRight); // %4X 
} 
```

**Comment by gavinlyonsrepo** [comment]
Hi 

I am aware of this typo.
I have tested it and the bug has no effect on program output as the enum  effected only has two 
values (AlignTextType_e).

I corrected it in commit number 741f351 ("version 1.9.1") in the repository, for model 1 
, forgot to do it for model 2, will do now. 
However that commit is not included in Version 1.9.0 (the latest release).
It will be  included if and when there is a new release.

Thank you
